tools/ats2-env.sh

```diff
@@ -86,7 +86,7 @@ case $ddir in
       unset CPATH
       unset LD_LIBRARY_PATH
       unset LIBRARY_PATH
-      run "module load draco/xl2020.08.19-cuda-11.0.2"
+      run "module load draco/xl2020.11.12-cuda-11.2.0-beta"
```

Comment:
@KineticTheory Eventually we will remove the beta?

Reply:
This beta gave errors in the volta builds, can we stick with 11.0.2?

Reply:
Yes.  LLNL is rapidly incrementing the recommended toolchains.  Hopefully this will stabilize soon.

Reply:
> 
> 
> This beta gave errors in the volta builds, can we stick with 11.0.2?

We can stick with 11.0.2.  This content of this script doesn't really matter except for my release testing.
Also:
* As Jae mentioned, 11.2.0 is also available on ATS-2.
* And, the preferred/default compiler is now xl/2021.03.11.
* I'll bump the environments to the new toolchain in a couple of weeks.

Reply:
I'll test the regressions with 11.2.0 right now to see if they fixed the issues in the beta.